homeassistant/components/homematicip_cloud/.translations/en.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "title": "HomematicIP Cloud",
+    "step": {
+      "init": {
+        "title": "Pick HomematicIP Accesspoint",
+        "data": {
+          "accesspoint": "Accesspoint ID (SGTIN)",
+          "pin": "Pin Code (optional)",
+          "name": "Name (optional, used as name prefix for all devices)"
+        }
+      },
+      "link": {
+        "title": "Link Accesspoint",
+        "description": "Press the blue button on the accesspoint and the submit to register HomematicIP with Home Assistant.\n\n![Location of button on bridge](/static/images/config_philips_hue.jpg)"
```

Comment:
Ehr, this is the phlips hue image…

Reply:
Thanks for your review... where do I need to place the picture of the HomematicIP access point?

Reply:
@MartinHjelmare Can you point me to a location where I have to put the picture? 
Thanks Mattias

Reply:
I think it's here:
https://github.com/home-assistant/home-assistant-polymer/tree/master/public/images

Reply:
Thanks, PR created